### PR TITLE
script to subscribe all first members. Auto subscribe new members to group.expense.created

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: scripts/subscribe_to_group.expense.created.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import models from '../server/models';
+
+const debug = require('debug')('subscribe');
+
+const {
+  Notification,
+  UserGroup
+} = models;
+
+const processRows = (rows) => {
```
Comment (asood123) [inline scripts/subscribe_to_group.expense.created.js]: fyi, don't need processRows function. can add promise.map directly in line 31

File: scripts/subscribe_to_group.expense.created.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+};
+
+const init = () => {
+
+  const query = {
+      where: {
+        role: 'MEMBER'
+      },
+      order: [['id', 'ASC']],
+      limit: 20
```
Comment (asood123) [inline scripts/subscribe_to_group.expense.created.js]: This is only going to fetch 20 rows. Don't we want to add this for all groups?

Comment (xdamman) [inline scripts/subscribe_to_group.expense.created.js]: argh of course that was for testing. good catch.

**Comment by xdamman** [comment]
deployed and executed the script

**Pull request merged**